src/neuroglancer/help/input_event_bindings.ts

```diff
@@ -151,7 +151,14 @@ export class InputEventBindingHelpDialog extends SidePanel {
         layerBindings = [];
         layerToolBindingsMap.set(tool.layer, layerBindings);
       }
-      layerBindings.push([`shift+key${key.toLowerCase()}`, tool.description]);
+      const globalBindings = [...bindings][0][1].parents[0];
```

Comment:
Rather than iterate over all bindings for each tool, it would be better to build a Map<string, string[]> of bindings for each tool (i.e. where the action matches "^tool-[A-Z]$") outside the loop.

You can iterate over all entries of an EventActionMap (including any parents) using the `entries()` method which returns an iterator.